pkg/testing/integration/program.go

```diff
@@ -311,8 +311,8 @@ type ProgramTestOptions struct {
 	// uses Node, Python, .NET or Go.
 	PrepareProject func(*engine.Projinfo) error
 
-	// Array of dependencies which come from local packages.
-	LocalDependencies []LocalDependency
+	// Array of provider plugin dependencies which come from local packages.
+	LocalProviders []LocalDependency
```

Comment:
Code Search suggests this is an OK change 👍 

https://cs.github.com/?scope=org%3Apulumi&scopeName=pulumi&q=LocalDependencies

Reply:
Yeah, I was going to say, that Fraser added this very recently, so I doubt anyone (even outside of Pulumi) has used it yet. So I'm good with the rename.